tests/e2e/adminCourseManagement.spec.ts

```diff
@@ -198,8 +198,9 @@ describe("Course creation, edition and deletion", () => {
     });
 
     it("Can edit modules", () => {
-        cy.get(`input[id=check_module_${course.moduleIds[0]}]`).uncheck();
-        cy.get(`input[id=check_module_${newModule}]`).check();
+        cy.get("span").contains(course.moduleIds[0]).get("i[id='remove']").click();
```

Comment:
if you choose to remove the non-unique id on the tags, then use a class to identify the correct element

Reply:
done

Reply:
done